Source/WebCore/html/HTMLButtonElement.h

```diff
@@ -40,6 +40,9 @@ class HTMLButtonElement final : public HTMLFormControlElement {
     
     const AtomString& value() const;
 
+    const AtomString& command() const;
+    void setCommand(const AtomString& value);
```

Comment:
Seems unfortunate that we have to write the `setCommand` function even though it’s what `[Reflect]` would have generated. Seems a common pattern to need getter logic, but have the setter side be standard reflection, and I’m surprised we never made a version of `Reflect` for that. Since we do have to write the function, I'll note that the setter doesn’t need to name its argument here:
```suggestion
    void setCommand(const AtomString&);
```

Reply:
Yeah a ReflectSetter idl attribute would be very nice and I imagine probably not too much work on the binding side. I know one of the other engines has it (can't think which off the top of my head)

Reply:
I looked over the bindings implementation and indeed it would be an easy project. I might do it soon.